File: C++/class bank.cpp
#include<iostream>
using namespace std;
class bank
{
	public:
		int accno, balance, amt;
		int getdeposite()
		{
			cout<<"Enter deposite amount :";
			cin>>amt;
			balance=balance+amt;
			cout<<"Total Amount after Deposite :"<<balance<<endl;
			return 0;
		}
		int getwithdrow()
		{
			cout<<"Enter withdrow amount :";
			cin>>amt;
			balance=balance-amt;
			cout<<"Total Amount after withdrow:"<<balance<<endl;
		}
		
};
int main()
{
	bank b1;
	cout<<"Enter account no and balance";
	cin>>b1.accno>>b1.balance;
	b1.getdeposite();
	b1.getwithdrow();
	return 0;
}

File: Csharp/class_bank.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace class_bank
{
    class Program
    {
        class Bank
        {
            string name;
            int acno;
            int bal = 1000;
            int withdrawl, deposite, amt;
            public void getdata(string name,int acno,int amt)
            {
                this.name = name;
                this.acno = acno;
                deposite = amt + bal;
                withdrawl = bal - amt;
            }
            public void displaydata()
            {
                Console.WriteLine("Name : " + name);
                Console.WriteLine("Acc No : " + acno);
                
                Console.WriteLine("After deposite : " + deposite);
                Console.WriteLine("After withdrawl : " + withdrawl);
            }
        }
        static void Main(string[] args)
        {
            Bank b = new Bank();
            Console.WriteLine("Enter name & acc no :");
             string n = Console.ReadLine();
            int a = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Enter Amount :");
            int am = Convert.ToInt32(Console.ReadLine());
            b.getdata(n, a, am);
            b.displaydata();
            Console.ReadLine();

               



        }
    }
}
